pynvim/plugin/script_host.py

```diff
@@ -82,7 +82,7 @@ def python_execute(self, script, range_start, range_stop):
         """Handle the `python` ex command."""
         self._set_current_range(range_start, range_stop)
 
-        if script[0] == '=':
+        if script.startswith('='):
```

Comment:
(script can't be an empty string here when called via rpc / `:python3` command, but it'd be a more error-safe code)